tests/src/test_app.py

```diff
@@ -121,18 +121,18 @@ def test_mapathon_total_contributor_mapathon_query_builder():
ashtags\') ~~ \'%hotosm-project-11210;%\' OR ("tags" -> \'hashtags\') ~~ \'%hotosm-project-10985;%\' OR ("tags" -> \'hashtags\') ~~ \'%hotosm-project-10988;%\' OR ("tags" -> \'hashtags\') ~~ \'%hotosm-project-11190;%\' OR ("tags" -> \'hashtags\') ~~ \'%hotosm-project-6658;%\' OR ("tags" -> \'hashtags\') ~~ \'%hotosm-project-5644;%\' OR ("tags" -> \'hashtags\') ~~ \'%hotosm-project-10913;%\' OR ("tags" -> \'hashtags\') ~~ \'%hotosm-project-6495;%\' OR ("tags" -> \'hashtags\') ~~ \'%hotosm-project-4229;%\' OR ("tags" -> \'hashtags\') ~~ \'%mapandchathour2021 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11224 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10042 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-9906 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-1381 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11203 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10681 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-8055 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-8732 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11193 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-7305 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11210 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10985 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10988 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-11190 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-6658 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-5644 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-10913 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-6495 %\' OR ("tags" -> \'comment\') ~~ \'%hotosm-project-4229 %\' OR ("tags" -> \'comment\') ~~ \'%mapandchathour2021;%\')\n    ),\n    T2 AS (\n        SELECT (each(tags)).key AS feature,\n            user_id,\n            username,\n            count(distinct id) AS count\n        FROM osm_element_history AS t2, t1\n        WHERE t1.changeset_id    = t2.changeset\n        GROUP BY feature, user_id, username\n    ),\n    T3 AS (\n        SELECT user_id,\n            username,\n            SUM(count) AS total_buildings\n        FROM T2\n        WHERE feature = \'building\'\n        GROUP BY user_id, username\n    )\n    SELECT user_id,\n        username,\n        total_buildings,\n        public.tasks_per_user(user_id,\n            \'11224,10042,9906,1381,11203,10681,8055,8732,11193,7305,11210,10985,10988,11190,6658,5644,10913,6495,4229\',\n            \'2021-08-27T09:00:00\',\n            \'2021-08-27T11:00:00\',\n            \'MAPPED\') AS mapped_tasks,\n        public.tasks_per_user(user_id,\n            \'11224,10042,9906,1381,11203,10681,8055,8732,11193,7305,11210,10985,10988,11190,6658,5644,10913,6495,4229\',\n            \'2021-08-27T09:00:00\',\n            \'2021-08-27T11:00:00\',\n            \'VALIDATED\') AS validated_tasks,\n        public.editors_per_user(user_id,\n            \'2021-08-27T09:00:00\',\n            \'2021-08-27T11:00:00\') AS editors\n    FROM T3;\n    '
```

Comment:
here, we should have semicolon, not a space as it is ("tags" -> \'hashtags\') ~~